controllers/storagecluster/cephcluster.go

```diff
@@ -1707,3 +1718,52 @@ func getFullRatio(specifiedRatio *float64) *float64 {
 	defaultRatio := 0.85
 	return &defaultRatio
 }
+
+// updateRookCephOperatorHostNetwork updates the rook-ceph-operator CSV to enable host network
+func updateRookCephOperatorHostNetwork(ctx context.Context, c client.Client, opNamespace string) error {
+	csvList := &opv1a1.ClusterServiceVersionList{}
+	labelSelector := labels.SelectorFromSet(map[string]string{
```

Comment:
I don't thin you need to use SelectorFromSet here.  You only need to match a single label.